### DISCUSSION TITLE
Do saleor support mysql?

### DISCUSSION THREAD

**Discussion by restarea92** [saleor/saleor#12153] [Q&A] [answered]
...

**Comment by karolkielecki** [comment, marked as answer]
Hi @restarea92, Saleor is built and optimized for PostgreSQL. You need to fork the code and change Saleor's logic from PostgreSQL to MySQL. However, please bear in mind that forking the code is not recommended because that can lead to technical debt and issues with maintaining and updating to the latest versions.
